Userland/Libraries/LibCrypto/PK/RSA.h

```diff
@@ -31,8 +31,6 @@ class RSAPublicKey {
     {
     }
 
-    //--stuff it should do
```

Comment:
Thanks for getting rid of the useless comment :P

Reply:
:D